gulpfile.js

```diff
@@ -43,20 +31,7 @@ gulp.task('lint', gulp.parallel('lint-html', 'lint-js'));
 gulp.task('unittest', unittestTask);
 gulp.task('test', gulp.parallel('lint', 'unittest'));
 gulp.task('watch', watchTask);
-gulp.task('default', gulp.parallel('lint', 'build', 'watch'));
-
-function rollupTask(done) {
-  run('rollup/dist/bin/rollup', ['-c'], done);
-}
-
-/**
- * Copy the files from `/dist` to the root directory.
- * @todo remove at version 1.0
- */
-function copyDistFilesTask() {
```

Comment:
removing this will break `gulp package` so that the release process doesn't work. you might want to leave this but make it run as the first step of `package` instead of the last step of `build`